README.MD

```diff
@@ -1,42 +1,43 @@
-#### Docker setup:  
-```
-sudo systemctl start docker  
-sudo usermod -a -G docker $USER  
-sudo reboot  
-docker pull dreaserdocker/dakar
-docker-compose pull
-docker login
-```
+# Dakar
+
+Fully reactive online travel agency made with Spring Boot 2 (w/ WebFlux), Angular, GraphQL, CouchBASE, Maven, CircleCI, and Docker.  
+
+The goal is to learn as much as possible about new tech while creating a small but hopefully helpful travel agency fully made in reactive programing.
+
+## Maven Central Repo
+https://mvnrepository.com/artifact/com.dakar/dakar
+
+## Docker hub 
+https://hub.docker.com/repository/docker/dakarinternational/dakar
 
+## Github Release
+https://github.com/DaKarInternational/DaKar/releases/tag/3.0.0
 
-#### On way to launch the app in local: 
+
+### On way to launch the app  
 1. build jar  
-`mvnw install -DskipTests  `
+`mvnw clean install  `
```

Comment:
Will the jar be created if we do not skip the tests as the database is not launched?